vendor/lowrisc_ibex.lock.hjson

```diff
@@ -1,4 +1,4 @@
-// Copyright lowRISC contributors.
+// Copyright lowRISC contributors (OpenTitan project).
```

Comment:
Is this change intended?

Reply:
I think it comes from the OpenTitan vendor.py script, so not quite correct but I think important we leave the files are they were generated from vendor tooling, we can maybe add some flag to that tool so you suppress this addition.